frontend/src/modules/integration/integration-service.js

```diff
@@ -135,6 +135,26 @@ export class IntegrationService {
     return response.data;
   }
 
+  static async githubMapRepos(integrationId, mapping, segments) {
+    const tenantId = AuthCurrentTenant.get();
+    const response = await authAxios.put(
+      `/tenant/${tenantId}/integration/${integrationId}/github/repos`,
+      {
+        mapping,
+        segments,
+      },
+    );
+    return response.data;
+  }
+
+  static async fetchGitHubMappings(integrationId, segments) {
```

Comment:
Are you using this `segments` anywhere? Was it supposed to go as params in get request?

Reply:
Fixed